site-src/v1alpha2/references/cross-namespace-references.md

```diff
@@ -0,0 +1,273 @@
+In particular, it's expected that Gateways and Routes may exist in different namespaces,
+or that Services may be referred to by Routes in a another namespace.
+
+However, this significantly violates the idea of a namespace as the edge of a trust domain.
+In order to bring cross-namespace references under the control of the owner of the referent object's namespace,
+the Gateway API has a ReferencePolicy object that must be created in the referent namespace for the reference to be successful.
+
+To put this another way, if an object is referred to from outside its namespace,
+the object's owner must create a ReferencePolicy object that describes how that reference is allowed.
+This page explains how this process works.
+
+In the past, we've seen that forwarding traffic across namespace boundaries is a desired feature,
+but without the kinds of safeguards proposed here, [vulnerabilities](https://github.com/kubernetes/kubernetes/issues/103675)
+can emerge.
+
+## ReferencePolicy
+
+To ensure that Gateway API is able to safely provide this functionality,
+we need to enforce a handshake mechanism that requires resources in both namespaces to agree to this reference.
+To accomplish that, a ReferencePolicy resource has been be introduced.
+
+![Reference Policy](images/referencepolicy.png)
+
+With this model, Routes are able to directly reference Routes and Services in other namespaces.
+These references are only considered valid if a ReferencePolicy in the target namespace explicitly allows it.
+
+The following example shows how a HTTPRoute in namespace `foo` can reference a Service in namespace `bar`.
+In this example a ReferencePolicy in the `bar` namespace explicitly allows references to Services from HTTPRoutes in the `foo` namespace.
+
+```yaml
+kind: HTTPRoute
+metadata:
+  name: foo
+  namespace: foo
+spec:
+  rules:
+  - matches:
+    - path: /bar
+    forwardTo:
+      backend:
+      - name: bar
+        namespace: bar
+---
+kind: ReferencePolicy
+metadata:
+  name: bar
+  namespace: bar
+spec:
+  from:
+  - group: networking.gateway.k8s.io
+    kind: HTTPRoute
+    namespace: foo
+  to:
+  - group: core
+    kind: Service
+```
+
+### API design decisions
+This proposed API is fairly straightforward, but comes with a few notable decisions:
+
+1. Each ReferencePolicy only supports a single From and To section.
+   Additional trust relationships must be modeled with additional ReferencePolicy resources.
+1. Resource names are intentionally excluded from this policy for simplicity and because they rarely provide any meaningful protection.
+   A user that is able to write to resources of a certain kind within a namespace can always rename
+   resources or change the structure of the resources to match a given policy.
+1. A single Namespace is allowed per "From" struct.
+   Although a selector would be more powerful it may encourage unnecessarily insecure configuration.
```

Comment:
I don't mind being more assertive here and seeing how it plays out in the community.
```suggestion
   Although a selector would be more powerful, it encourages unnecessarily insecure configuration.
```